### ISSUE TITLE
create alt text for all amheida images

### ISSUE THREAD

**Issue by paregorios** [isawnyu/isaw.web#407]
Arbitrary manual checking reveals that some images in the Amheida section have good alt text, but others just have the names of the image files. The latter condition is a WCAG violation and must be corrected before the Amheida materials can be published.

Steps:

 - [ ] determine which images need alt text captioning (make a list)
 - [ ] get alt text from the amheida team
 - [ ] insert alt text through the plone (ideally the amheida team will do this themselves)

**Comment by paregorios** [comment]
Assigned this to myself to do the first step: make a list of all amheida-related images and whether they need alt text or not. Once this is done, we can publish all portions that don't include PDFs or videos.

**Comment by paregorios** [comment]
It looks like there are about 70 images in the amheida portion of the site: https://isaw.nyu.edu/members/te20-40nyu.edu/amheida-images/

**Comment by paregorios** [comment]
##Page-by-page inventory

- Acknowledgements
- Bibliography
- CUNY Excavations
- Database
- Dig House Library
- Directory
- El-Deir Partnership
- Excavations at Amheida
- How to Help
- Local Community
- Maps and Drawings
- Members of the Amheida Team, 2004-2015
- Publications
- Reports
- Satellite Image Enhancements
- Useful Links
- Videos

**Comment by skleinfeldt** [comment]
> It looks like there are about 70 images in the amheida portion of the site: https://isaw.nyu.edu/members/te20-40nyu.edu/amheida-images/. What the collection here doesn't do is tell me where these are surfaced and whether alt text has been applied in those locations.

FYI @paregorios - there were many unused images on the site but I don't think I uploaded any of them. Aside from the plans, maps, and satellite areas I cherry picked the images that were actually used on the homepage, CUNY Excavations, El-Deir Partnership, and Local Community pages and uploaded them manually. IIRC there is a file in the  on the original site in the inc/img/sat area that contains descriptions of the satellite images (or maybe I am thinking of the maps and plans) but I think those descriptions were also present as text on the page.

Bummer about all the alt text work...

**Comment by skleinfeldt** [comment]
And I meant to say - I did not do anything about alt text when moving stuff as I had no information on the images (aside from that text file info and the captions which were already on the relevant pages.)
